index.rst

```diff
@@ -326,33 +326,157 @@ Analysis of unbinned data is clearly needed for pipeline data quality analyses,
+
+The Visit database implementation is left for future developers, but similarly to the Butler registry, a PostgreSQL database, appears to be a viable option.
+
+Again we leave the implementation (e.g., replication vs. reprocessing) to future developers.
+Still, it is a strict requirement that the contents be identical in both instances, with the nominal retention period of 30 days at the Summit.
+
+**Visit and detector tables**
+
+The image headers and the Rapid Analysis outputs are the primary sources of information for the visit and detector tables.
+
+Rapid Analysis will have access to the headers during processing and can consolidate the raw image metadata into per-detector, and per-visit datasets along with the per-visit summary statistics from SFP persisted as Butler datasets.
+Scalar metrics (see :ref:`analysis tools overview <analysis_tools_overview>`) can also be persisted as Butler datasets (see `DMTN-203 Tracking Metrics in Butler`_).
+
+A mechanism for creating these tables in a relational database already exists in DM.
+In particular, if the schema for a Butler dataset is defined in the `Science Data Model`_ (SDM) repository, the database schema can be created from those definitions.
+
+Finally, the method for loading the Rapid Analysis results must ensure the data is readily available in the Visit database after each visit is processed.
+The current method used for DRP based on the extraction of datasets from the Butler and loading into a database after the processing will not work for the
+Rapid Analysis use case.
+
+**Telemetry summary tables**
+
+The current approach for combining visit information with telemetry data is a "join on time," which essentially involves querying two databases, the Butler registry to retrieve the visit's start and end times and the EFD to retrieve the telemetry values in that time range.
+
+The database `use cases`_ Confluence page indicates the telemetry information that would be most useful in the Visit database.
+Typically the full time series from the EFD is not required and should be resampled onto the visit time range via summary statistics.
+The information, usually spread across multiple EFD topics, should also be consolidated into fewer tables to make it easier to query and join with other tables in the Visit database.
+
+A tool for creating these "telemetry summary tables" in a relational database does not currently exist.
+The `SQR-58 The EFD transformation service`_ presents a possible solution by filtering and combining information from multiple EFD topics and aggregating values over time.
+This tool must be flexible enough to incrementally add new tables or add new columns to existing tables in the Visit database as needed.
+For the Rapid Analysis use case, loading past data into the new tables/columns is not required after evolving the schema at the Summit.
+That's compatible with Rapid Analysis being run once at the Summit.
```

Comment:
Consider:
For the Rapid Analysis use case, loading past data into the new tables/columns is not required after evolving the schema at the Summit because Rapid Analysis is only to be run once at the summit.